tests/checkers/inference/test/CFInferenceTest.java

```diff
@@ -39,6 +39,10 @@ public String getPathToAfuScripts() {
         return System.getProperty("path.afu.scripts");
     }
 
+    public String getPathToInferenceScript() {
+        return "./scripts/inference";
```

Comment:
Ok, that still would be better since we don't need to change the source code. All we need is changing the config file(build script), which is centralized place for all the configurations.